apps/evm/src/containers/PrimeStatusBanner/index.tsx

```diff
@@ -422,7 +422,9 @@ const PrimeStatusBanner: React.FC<PrimeStatusBannerProps> = props => {
     token: xvs,
   });
 
-  const highestPrimeSimulationApyBoostPercentage = primeOrderedApys[0];
+  const highestPrimeSimulationApyBoostPercentage = primeOrderedApys
+    ? primeOrderedApys[0]
+    : undefined;
```

Comment:
Out of curiosity, why did you update this logic?

While the Prime APYs are loading, the banner would now display but with the placeholder character "-" in place of the highest APY.